perun-base/src/main/java/cz/metacentrum/perun/taskslib/model/Task.java

```diff
@@ -88,22 +90,38 @@ public void setRecurrence(int recurrence) {
 		this.recurrence = recurrence;
 	}
 
-	public Date getStartTime() {
+	public LocalDateTime getStartTime() {
 		return startTime;
 	}
 
-	public void setStartTime(Date startTime) {
+	public long getStartTimeAsLong() {
+		return startTime.atZone(ZoneId.systemDefault()).toEpochSecond();
```

Comment:
Please do check on ``null`` for all such getters. I believe we previously returned ``null`` in such case, because object was a ``Date``, now we return primitive ``long`` which isn't an object. We should probably return ``Long`` so its serialized either as ``null`` or ``long`` value.

Reply:
This comment was not resolved last time. Please do check on ``null`` before you call method on the object.